Dockerfile.rocm

```diff
@@ -38,10 +38,9 @@ FROM fetch_vllm AS build_vllm
 ARG USE_CYTHON
 # Build vLLM
 RUN cd vllm \
-    && python3 -m pip install -r requirements-rocm.txt \
-    && python3 setup.py clean --all  \
+    && python3 -m pip install build \
```

Comment:
i'm new, but why the difference in building here?

Reply:
The tl;dr is that using `setup.py bdist_wheel` [is deprecated](https://packaging.python.org/en/latest/discussions/setup-py-deprecated/)